mobile/src/bundles/talent/components/skills-and-projects-form/skills-and-projects-form.tsx

```diff
@@ -57,6 +62,10 @@ const SkillsAndProjectsForm: React.FC<Properties> = ({
         void handleSubmit(onSubmit)();
     }, [handleSubmit, onSubmit]);
 
+    useEffect(() => {
+        void dispatch(getHardSkillsData());
+    }, [dispatch]);
+
```

Comment:
I guess here we also need loader

Reply:
on this screen

Reply:
its form, that inside mobile/src/bundles/talent/screens/skills-and-projects/skills-and-projects.tsx